src/components/StepsList/index.jsx

```diff
@@ -99,36 +90,32 @@ const StepsList = ({
     setVisible(displayToolTip());
   }, [activeStep, availableSteps, hovering]);
 
+  const handleStepClick = (index) => {
+    setActiveStep(index);
+  };
+
   return (
     <Box sx={{ color: 'primary.text' }}>
-      <Stepper activeStep={activeStep} alternativeLabel nonLinear>
-        {calculatorList.map((label, index) => (
-          <Step
-            key={label}
-            disabled={maxAvailableStep + 1 < index}
-            sx={{
-              '& .MuiSvgIcon-root': {
-                color: maxAvailableStep + 1 < index ? '' : '#4f5f30',
-                '&.Mui-active': {
-                  color: '#77b400',
-                },
+      <PSAStepper
+        steps={calculatorList}
+        strokeColor="#fffff2"
+        maxAvailableStep={maxAvailableStep}
+        onStepClick={(tab, index) => handleStepClick(index)}
+        boxProps={{ sx: { marginBottom: '1rem' } }}
+        stepperProps={{ activeStep }}
+        stepButtonProps={(step, index) => ({
```

Comment:
This will not work for the step button props. Maybe you can put the data-test to the stepper in share components

Reply:
We had data-test as prop in the StepButton that's why I've passed it here as step button prop

Reply:
But you're passing it as a function here so it'll not apply to each step buttons